helm/charts/hydra-maester/values.yaml

```
@@ -32,6 +32,12 @@ adminService:
   # -- Set the clients endpoint, should be `/clients` for Hydra 1.x and
   # `/admin/clients` for Hydra 2.x
   endpoint: /admin/clients
+  # -- Scheme used by Hydra client endpoint. May be "http" or "https"
+  scheme:
+  # -- TLS ca-cert path for hydra client
+  tlsTrustStorePath:
```

```suggestion
  tlsTrustStorePath: ""
```
afaik an empty string should be evaluated to `false` in the if clause, but need to check it :)